examples/README.md

```diff
@@ -0,0 +1,11 @@
+# Examples
+
+## Project setup
+```
+composer install
+```
+
+### Run example as phpunit test
+```
+php vendor/phpunit/phpunit/phpunit examples
```

Comment:
Same here: from examples subdir or project root?

Reply:
Added clarification in README.md